content/en-us/reference/engine/classes/AssetService.yaml

```
@@ -75,10 +75,12 @@ methods:
     writeCapabilities: []
   - name: AssetService:CreateAssetAsync
     summary: |
-      Creates a new asset from the given object.
+      Uploads a new asset to Roblox from the given object.
     description: |
-      Creates a new asset from the given object in Plugin or Open Cloud Luau
-      Execution context.
+      Uploads a new asset to Roblox from the given object.
+
+      This method can only be used in locally loaded plugins and uploads
+      assets without prompting first.
```

I think this is valuable information, i would like to add that this is temporary (even though it has taken much longer than I would have like to get support for published plugins :( ).

I'm not too familiar with our docs standards, does something like this make sense?
```suggestion
      (Temporary) For now, this method can only be used in locally loaded plugins and uploads
      assets without prompting first.
```